sound_byte/lib/pages/friendList.dart

```diff
@@ -4,4 +4,99 @@ import 'package:sound_byte/services/authentication.dart';
 import 'package:sound_byte/pages/userProfile.dart';
 import 'package:sound_byte/pages/friendProfile.dart';
 import 'package:sound_byte/pages/friendScreen.dart';
+import 'package:sound_byte/pages/user.dart';
 
+class FriendList extends StatelessWidget{
+  User user;
+  List<String> user_friends;
+  int number_of_friends;
+  String user_id;
+
+  FriendList (User user) {
+    this.user = user;
+    this.user_friends = user.friends;
+    this.number_of_friends = user.friends.length();
+    this.user_id = user.userId;
+  }
+
+  FriendList.nullFriendList() {
+    this.user = new User("Sound Byte", 
```

Comment:
there is a null user object you should have used use "new User.null();"

Reply:
I instead took off the nullfriends list method and initialized the object.